sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsErrorTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Pipeline;
+using Azure.Core.Testing;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Azure.AI.TextAnalytics.Tests
+{
+    public class TextAnalyticsErrorTests : SyncAsyncPolicyTestBase
+    {
+        public TextAnalyticsErrorTests(bool isAsync) : base(isAsync) { }
+
+        [Test]
+        public async Task ConvertTextAnalyticsErrorToException()
+        {
+            var mockResponse = new MockResponse(200);
+            TextAnalyticsError error = new TextAnalyticsError("TestErrorCode", "TestMessage", "TestTarget");
+            var exception = await mockResponse.CreateRequestFailedExceptionAsync(error);
+
+            Assert.AreEqual("TestErrorCode", exception.ErrorCode);
+            StringAssert.Contains("TestMessage", exception.Message);
+            StringAssert.Contains("TestTarget", exception.Message);
```

Comment:
Should this be `exception.target` ?

Reply:
I don't believe so.  We expose Target on TextAnalyticsError, but not on RequestFailedException.  For the exception, we just want to validate that the target string made it into the message.  This is because we don't currently know of a scenario where users would do something different programmatically based on this field.